storage/simulation/cluster.go

```diff
@@ -64,6 +69,8 @@ func createCluster(stopper *stop.Stopper, nodeCount int) *Cluster {
 		nodes:         make(map[proto.NodeID]*Node),
 		stores:        make(map[proto.StoreID]*Store),
 		ranges:        make(map[proto.RangeID]*Range),
+		rand:          rand,
+		seed:          seed,
```

Comment:
You're going to have to be careful in the simulator implementation to make it deterministic. For example, no goroutines which are independently generating and acting upon random numbers.